### DISCUSSION TITLE
我为啥无账户密码运行

### DISCUSSION THREAD

**Discussion by flyingnight** [yuudi/rclone-webui-angular#49] [Q&A]
我使用参数rcd --rc-web-gui --rc-web-gui-update --rc-web-fetch-url="https://s3.yuudi.dev/rwa/embed/version.json" --rc-web-gui-no-open-browser --rc-serve --rc-addr :5572 --rc-web-gui-force-update
但是打开webui时，还是需要验证账户密码，同时，因为我没有设置过，就无法进入了，要怎么样才能无密码进入呢

**Comment by yuudi** [comment]
这是rclone的行为，默认情况下会自动创建一个密码并使用这个密码打开浏览器，但当你使用了`--rc-web-gui-no-open-browser`时，不会自动打开浏览器。

你可以考虑使用`--rc-user="<your username>" --rc-pass="<your password>"`手动设置密码，或者`--rc-no-auth`无密码使用
